Content.Client/Power/APC/ApcVisualizerSystem.cs

```diff
@@ -78,11 +78,11 @@ enum ApcVisualLayers : byte
     /// <summary>
     /// The sprite layer used for the interface lock indicator light overlay.
     /// </summary>
-    InterfaceLock = 0,
+    InterfaceLock = 3,
     /// <summary>
     /// The sprite layer used for the panel lock indicator light overlay.
     /// </summary>
-    PanelLock = 1,
+    PanelLock = 4,
     /// <summary>
     /// The first of the lock indicator light layers.
     /// </summary>
```

Comment:
This is really scuffed and you should just fix the code to use the layer mappings properly.

Reply:
I changed everything to use the layer mapping system